src/openapi/paths/baseFieldLocalization.json

```diff
@@ -0,0 +1,77 @@
+{
+	"put": {
+		"operationId": "upsertBaseFieldLocalization",
+		"summary": "Creates or updates a base field localization.",
+		"tags": ["Base Fields"],
+		"security": [
+			{
+				"auth": []
+			}
+		],
+		"parameters": [
+			{
+				"name": "baseFieldId",
+				"description": "The PDC-generated ID of a base field.",
+				"in": "path",
+				"required": true,
+				"schema": {
+					"type": "integer"
+				}
+			},
+			{
+				"name": "language",
+				"description": "The valid IETF language tag for a base field localization",
```

Comment:
Again unrelated to this PR, but I think a link to the list of valid names here would help.